homeassistant/components/telegram_bot/services.yaml

```diff
@@ -251,6 +251,24 @@ send_video:
       example: "/path/to/the/video.mp4"
       selector:
         text:
+    thumb_url:
```

Comment:
Services do support now be [translatable](https://developers.home-assistant.io/blog/2023/07/11/translating-services). For consistency, please adopt the new service attributes to be translatable too.